addons/portal/data/mail_templates.xml

```diff
@@ -6,7 +6,9 @@
             <p>Dear <span t-esc="partner.name"/>,</p>
             <p>You have been invited to access the following document:</p>
             <br/>
-            <a t-attf-href="#{share_link}" style="background-color: #875A7B; padding: 10px; text-decoration: none; color: #fff; border-radius: 5px; font-size: 12px;"><strong>Open </strong><strong t-esc="record.display_name"/></a><br/>
+            <div style="margin-top: 1rem;">
```

Comment:
Wouldn't it be the same to just add this margin to the `<a>` tag? That way we have less diff.